src/app_engine/apprtc.py

```diff
@@ -122,11 +122,13 @@ def get(self):
 
 app = webapp2.WSGIApplication([
     ('/', MainPage),
+    # TODO(jiayl): Remove the deprecated API when Android is updated.
```

Comment:
nit: move the deprecated stuff to the bottom so it's clear which ones are to be removed